runner-config.md

```diff
@@ -0,0 +1,44 @@
+To run the tests with a customized configuration, you can add a new setting with key named: `java.test.config`.
+
+> [How to create new settings in VS Code?](https://code.visualstudio.com/docs/getstarted/settings#_creating-user-and-workspace-settings)
+
+## TL;DR
+Here is an example of the configuration schema:
+```js
+"java.test.config": [
+    {
+        "name": "myConfiguration"
+        "workingDirectory": "${workspaceFolder}"
+        "args": [ "-c", "com.test" ],
+        "vmargs": [ "-Xmx512M" ],
+        "env": { "key": "value" },
+    },
+    {
+        // Another configuration item...
+    },
+  ...
+]
+```
+
+> Note: The first configuration item in the array will be used as the default configuration when you trigger `Run Test` or `Debug Test`
```

Comment:
Having second thought regarding this. If a test configuration exists, then users have no way of running without a config. It's a risky move. I would suggest that we leave `Run Test` and `Debug Test` alone.